msal/src/Microsoft.Identity.Client/Features/ConfidentialClient/ConfidentialClientApplication.cs

```diff
@@ -38,7 +38,8 @@ namespace Microsoft.Identity.Client
     /// <summary>
     /// Class to be used for confidential client applications like Web Apps/API.
     /// </summary>
-    public sealed class ConfidentialClientApplication : ClientApplicationBase, IConfidentialClientApplication
+    public sealed class 
```

Comment:
>  [](start = 23, length = 1)

This is not needed. I think the width we have on our screens these days are ok with this. to me this is reducing readability

Reply:
Oops, I was playing with .editorconfig to enforce styles and VS changed the formatting. I'll revert.